src/openforms/registrations/contrib/zgw_apis/plugin.py

```diff
@@ -52,6 +52,12 @@ class ZaakOptionsSerializer(JsonSchemaSerializerMixin, serializers.Serializer):
             "Indication of the level to which extend the dossier of the ZAAK is meant to be public."
         ),
     )
+    medewerker_roltype = serializers.URLField(
+        required=False,
+        help_text=_(
+            "URL of the ROLTYPE to use for employees filling in a form for a citizen/company."
+        ),
+    )
```

Comment:
As mentioned above, make this a regular `CharField` where the omschrijving can be looked up, and validation should then try to retrieve it within the configured zaaktype. If it can't be found -> validation error.